tests/resources/fake_ext_plugin/pyproject.toml

```
@@ -0,0 +1,13 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "eodag_fakeplugin"
+version = "1.0"
+
+[project.entry-points."eodag.plugins.api"]
+FakePluginAPI = "eodag_fakeplugin:FakePluginAPI"
+
+[tool.setuptools]
+packages = ["eodag_fakeplugin"]
```

```suggestion
[tool.setuptools]
packages = ["eodag_fakeplugin"]

[tool.setuptools.package-data]
eodag_fakeplugin = ["providers.yml"]
```

This does not affect the test but is needed when manually installing this plugin out of this test